src/Microsoft.AspNetCore.SystemWebAdapters.CoreServices/RemoteAppClientExtensions.cs

```diff
@@ -50,27 +51,7 @@ public static ISystemWebAdapterBuilder AddRemoteAppClient(this ISystemWebAdapter
                 client.DefaultRequestHeaders.Add(options.ApiKeyHeader, options.ApiKey);
             });
 
-        configure(new Builder(builder.Services));
-
-        return builder;
-    }
-
-    public static ISystemWebAdapterRemoteClientAppBuilder Configure(this ISystemWebAdapterRemoteClientAppBuilder builder, Action<RemoteAppClientOptions> configure)
-    {
-        if (builder is null)
-        {
-            throw new ArgumentNullException(nameof(builder));
-        }
-
-        if (configure is null)
```

Comment:
Should probably keep this error check

Reply:
The null check is still there - this whole method was removed :)

Reply:
Ah, right. 👍